config/samples/olm_v1_clusterextension.yaml

```diff
@@ -281,5 +281,5 @@ spec:
   source:
     sourceType: Catalog
     catalog:
-      packageName: argocd-operator
+      packageName: foobar
```

Comment:
nit: looks like dev leftover?

Reply:
Oops! Yep.

Reply:
We do nothing related the name of the pkg right
Otherwise, it would fail. But good catcher 
It is good keep the name so we know the source (from where the mock data came from)